Jaswitha

Jaswitha opened the pull request:
changed naming conventions

Jaswitha committed:
added bean classes

Jaswitha committed:
modified

chasethenag420 commented on src/main/java/com/asu/cse545/group12/domain/Transaction.java:
```diff
@@ -0,0 +1,154 @@
+	private Date creationTimestamp;
+	
+	@Column(name = "TRANSACTION_TYPE")
+	private String transactionType;
```
what is difference between transaction_type and transaction_kind columns?

Jaswitha replied:
Hello,

transaction_kind will say if it is critical or non-critical.
transaction_type will say if it  is debit or credit .

Thanks
Jaswitha

On Thu, Oct 1, 2015 at 10:58 PM, Nagarjuna Myla notifications@github.com
wrote:

> In src/main/java/com/asu/cse545/group12/domain/Transaction.java
> https://github.com/chasethenag420/sbs/pull/13#discussion_r40993201:
> 
> > -   @GeneratedValue(strategy = GenerationType.IDENTITY)
> > -   @Column(name="TRANSACTIONID")
> > -   private int transactionId;
> >   +
> > -   @Column(name = "USERID")
> > -   private int  userId;
> > -   
> > -   @Column(name = "TRANSACTION_STATUS")
> > -   private String transactionStatus;
> > -   
> > -   
> > -   @Column(name = "CREATION_TIMESTAMP")
> > -   private Date creationTimestamp;
> > -   
> > -   @Column(name = "TRANSACTION_TYPE")
> > -   private String transactionType;
> 
> what is difference between transaction_type and transaction_kind columns?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/chasethenag420/sbs/pull/13/files#r40993201.

chasethenag420 commented on src/main/java/com/asu/cse545/group12/domain/UserPII.java:
```diff
@@ -0,0 +1,20 @@
+
+	@Column(name = "DATEOFBIRTH")
+	private Date DateOfBirth;
+	
```
No setter and getters?

Jaswitha committed:
added Dao interfaces and their implementations
Changed the name of User to Users as the name it is not taking .Let me
know if there are any changes to be updated.
